tests/installation/partitioning_raid.pm

```diff
@@ -42,7 +42,12 @@ sub addpart {
     send_key $cmd{donotformat};
     send_key "tab";
 
-    send_key_until_needlematch 'partition-selected-raid-type', 'down';
+    if ($size == 300 and get_var('UEFI')) {
```

Comment:
I updated this PR to use part names in addpart() instead of its size so now it is IMO much more self-explanatory.